hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/AbstractRootedOzoneFileSystemTest.java

```diff
@@ -274,6 +274,7 @@ void initClusterAndEnv() throws IOException, InterruptedException, TimeoutExcept
     userOfs = UGI_USER1.doAs(
         (PrivilegedExceptionAction<RootedOzoneFileSystem>)()
             -> (RootedOzoneFileSystem) FileSystem.get(conf));
+    random = new SecureRandom();
```

Comment:
`random` member variable is unused after d183df3fa874e7681d5fc5a29d03d4f55a2fe53c, please remove.  Also remove import of `SecureRandom`.